manager/testdata/new-blueprints/scenario5/data-quality.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: M4DModule
+metadata:
+  name: data-quality
+  namespace: m4d-system
+  labels:
+    version: 0.0.1
+    name: data-quality
+spec:
+  description: "This module deploys an arrow-flight service"
+  # this field is true if this module serves multiple users. Otherwise it
+  # is false. It is used for example when setting the endpoint to
+  # access the module service - if it is a multi-user module then
+  # the service name will be fixed and will be taken from
+  # M4DModule.Capabilities.Api.endpoint.host.
+  type: service   # deployed by fybrik
+  confChart:
+    name: ghcr.io/mesh-for-data/data-quality:0.1.0
+    values: {}
+  statusIndicators:
+  - kind: Deployment
+    successCondition: status == SUCCEEDED # TBD with correct values
+    failureCondition: status == FAILED # TBD with correct values
+    errorMessage: status.error # TBD with correct values
+  capabilities:
+  - capability: data-quality
+    scope: workflow
+    # No sink or target as prometheus scrapes metrics (configured outside)
+    # It may be possible that other notification sinks are configurable in
+    # different use cases
+  api:
+    endpoint:
+      port: 80 # Mandatory for each service
+    protocol: m4d-arrow-flight
+    dataformat: arrow
```

Comment:
We'll need to indicate that it supports plugins, in order for it to be able to call the Prometheus APIs.  Right?  I think we're missing the prometheus-client-config module too.